x-pack/plugin/security/qa/jwt-realm/src/javaRestTest/resources/jwk/README.md

```diff
@@ -0,0 +1,21 @@
+# Explanation of files in this directory
+
+`rsa-private-jwkset.json`, `rsa-public-jwkset.json`, `hmac-jwkset.json`
+-----------------------------------------------------------------------
+
+These files are created by running the tests in `JwtRealmGenerateTests`.
```

Comment:
JwtRealmGenerateTests does not have a test method to generate realm jwt3?

Reply:
It's there:
```
Issuer: jwt3-issuer
Audiences: jwt3-audience
Algorithms: HS512,HS384
HMAC JWKSet: {"keys":[{"kty":"oct","kid":"test-hmac-384","... }]}
```